src/index.ts

```diff
@@ -61,4 +63,15 @@ const applyGroupStage = (query: any, data: any) => {
   return results;
 };
 
+const applyProjectStage = (query: any, data: any) => {
+  return data.map((item: any) => {
+    const result: any = {};
+    for (let [key, value] of Object.entries(query)) {
+      if (value === 1) result[key] = item[key];
```

Comment:
I was reading the documentation for `$project` and noticed that it's possible to set this to `true`, `0` or `false`.  And I think `_id` is included, unless explicitly removed (though I could be wrong)

Reply:
That's true. I just added the inclusions as a starting point, I am working on adding the exclusion logic and will push the same soon. Thanks.